src/network/netdev/tunnel.c

```diff
@@ -897,7 +900,7 @@ const NetDevVTable ip6tnl_vtable = {
         .fill_message_create = netdev_ip6tnl_fill_message_create,
         .create_type = NETDEV_CREATE_STACKED,
         .config_verify = netdev_tunnel_verify,
-        .generate_mac = true,
```

Comment:
it would be good to explain why `generate_mac` is being removed in the commit message

Reply:
Sure. Done.